src/core/StringX.cpp

```diff
@@ -156,7 +155,7 @@ template<class T> void LoadAString(T &s, int id)
   cs.LoadString(id);
   s = cs;
 #else
-  s = _(core_st[id]);
+  s = core_st[id];
```

Comment:
This change breaks the translation of the text at runtime.

Reply:
I'll look into this. Just wanted to keep the core wx-free.